Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -99,7 +99,14 @@ def account_entry(person_object, custome_attributes):
     }
 
     for attr in custome_attributes:
-        account[attr] = person_object[attr]
+        try:
+            account[attr] = person_object[attr]
+        except Exception as e:
+            lower_cased_attr = attr[0].lower() + attr[1:]
+            if lower_cased_attr in person_object:
+                account[lower_cased_attr] = person_object[lower_cased_attr]
```

Comment:
This only accounts for the case where the user has accidentally capitalized the first letter of the attribute name. I'm not familiar with AD User attributes, but if they are unique (when indifferent to capitalization), I think we could make this more robust against capitalization typos - we can compare using `.casefold()` and assign the value of the index of the matching key.

Reply:
this is the standard in AD, but i will apply your suggestion in any case